CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 
 * Improve numeric inputs ([PR #1345](https://github.com/alphagov/govuk_publishing_components/pull/1345))
 * Enable custom classes and `aria-label` on button component ([PR #1344](https://github.com/alphagov/govuk_publishing_components/pull/1344))
+* Extend document-list component to show parts of a document ([PR #1326](https://github.com/alphagov/govuk_publishing_components/pull/1326))
```

Comment:
I don't think the rebase has worked properly here - it should be in 'unreleased' alongside the feedback component change that was merged yesterday

Reply:
Argh, thanks.